src/object/object_representation.c

```diff
@@ -800,25 +730,27 @@ or_mvcc_get_delid_chn (OR_BUF * buf, int mvcc_flags, int *error)
  * mvcc_rec_header(in): MVCC record header
  */
 STATIC_INLINE int
-or_mvcc_set_delid_chn (OR_BUF * buf, MVCC_REC_HEADER * mvcc_rec_header)
+or_mvcc_set_delid (OR_BUF * buf, MVCC_REC_HEADER * mvcc_rec_header)
 {
-  int error_code = NO_ERROR;
   assert (buf != NULL);
-
   ASSERT_ALIGN (buf->ptr, INT_ALIGNMENT);
-  if (mvcc_rec_header->mvcc_flag & OR_MVCC_FLAG_VALID_DELID)
-    {
-      /* MVCC DELID is active */
-      return or_put_bigint (buf, mvcc_rec_header->delid_chn.mvcc_del_id);
-    }
 
-  /* CHN is active */
-  error_code = or_put_int (buf, mvcc_rec_header->delid_chn.chn);
-  if ((error_code == NO_ERROR) && (mvcc_rec_header->mvcc_flag & OR_MVCC_FLAG_VALID_LONG_CHN))
+  if (!(mvcc_rec_header->mvcc_flag & OR_MVCC_FLAG_VALID_DELID))
     {
-      error_code = or_put_int (buf, 0);
+      return NO_ERROR;
     }
-  return error_code;
+
+  return or_put_bigint (buf, mvcc_rec_header->mvcc_del_id);
+
+}
+STATIC_INLINE int
```

Comment:
Add an empty line between the functions and remove lines 744 and 753(empty lines after return).